FirebaseAuth/Sources/Public/FirebaseAuth/FIRUser.h

```diff
@@ -19,13 +19,18 @@
 #import "FIRAuth.h"
 #import "FIRAuthDataResult.h"
 #import "FIRMultiFactor.h"
+#import "FIRPasskeyInfo.h"
 #import "FIRUserInfo.h"
 
 @class FIRAuthTokenResult;
 @class FIRPhoneAuthCredential;
 @class FIRUserProfileChangeRequest;
 @class FIRUserMetadata;
 @protocol FIRAuthUIDelegate;
+#if TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_OSX || TARGET_OS_MACCATALYST
```

Comment:
Should `TARGET_OS_VISION` be included here or excluded via availability in the header?

Reply:
visionOS is supported on 1.0+ beta, do you think we should include it here?

Reply:
Yes, we're striving to provide build support for Firebase for visionOS and community supported functionality. 

Note other examples of the usage of `TARGET_OS_VISION` so that it builds both on Xcode 14 and Xcode 15.

Reply:
Thank you Paul.  Will address this in a different PR for all occurance.  tracking ticket b/307595519